lib/archethic_web/controllers/api/schema/token_ledger.ex

```diff
@@ -10,7 +10,7 @@ defmodule ArchethicWeb.API.Schema.TokenLedger do
     embeds_many :transfers, Transfer do
       field(:to, Address)
       field(:amount, :integer)
-      field(:token, Address)
+      field(:tokenAddress, Address)
       field(:token_id, :integer)
```

Comment:
We should unify the `token_id` in camelCase for the API, as it's will be translated in snake case after casting

Reply:
yup that makes more senser, I was also thinking the same. Since atom parse does change it for node internal functions.